Eastern-Research-Group/expert-query
# Feature/105 check accessibility

## Related Issues:
* [EQ-105](https://jira.epa.gov/browse/EQ-105)

## Main Changes:
* Addressed accessibility concerns raised by _axe DevTools_ and the _WAVE Evaluation Tool_. I ignored the low contrast warnings of the input fields placeholder text, but one could argue that the color should be changed.
* Restructured the navigation buttons, so that the primary navigation links use the USWDS side navigation component. The glossary button is still a button, and the _User's Guide_ PDF link is positioned next to the _Contact Us_ link, since its behavior is closer to that of the contact page (new tab) and it's recommended to not put document links with primary navigation.
  * Clicking the **Expert Query** heading links to the root page, while clicking **Query ATTAINS Data** in the navigation links to "/attains". The root link is currently an alias for "/attains", but this might change in the future.
  * I don't know how I feel about the fact that the navigation relocates to the bottom of the page on tablet widths and smaller, but this seems to be the norm for many pages on the main EPA website.
* Simplified some logic on the home page.
* Added a level of responsiveness to the query form's grid, so that it varies from 1 to 3 columns as the width changes.

## Steps To Test:
1. Run the accessibility tools and verify that all pertinent issues have been addressed.
2. Test the new navigation layout on various screen sizes.
